inclusive-design/wecount.inclusivedesign.ca
# fix: correctly display the popup menu on the mobile layout (resolves #282)

## Description

Correctly display the popup menu on the mobile layout. The solution is to use different DOM structures for desktop and mobile layouts.

## Steps to test

1. Open any page in the desktop view and play with the navigation bar;
2. The menu items and search container should be displayed and functioned correctly;
3. Switch to the responsive design mode and select ipad view;
4. Click the menu button to open the popup menu;
5. The popup menu should be displayed right underneath the menu bar without any gap;
6. Open UIO;
7. Click the menu button to open the popup menu;
8. The popup menu should be displayed right underneath the menu bar;
9. On the mobile layout, other menu functionalities should function correctly.

**Expected behavior:** 

See above.
